### DISCUSSION TITLE
why setup function runs so long

### DISCUSSION THREAD

**Discussion by zhongxianmen2020** [pycaret/pycaret#2168] [Q&A]
Hi friends,

I am new to Pycaret and tried to run the tutorials for time series analysis using air passengers data

[pycaret](https://pycaret.gitbook.io/docs/learn-pycaret/official-blog/time-series-101-for-beginners

but the waiting time for the setup function is too long and the blue horizontal bar does not move. It has been there for almost one hour.  Does anyone know what happened?

Thanks,
Zhongxian

**Comment by ngupta23** [comment]
I think you have to confirm that the variables types are being detected correctly by pressing any key.

**Reply by zhongxianmen2020** [reply]
Thanks for the help. I did not do anything but just following the tutorials line by line. the setup function did not work

**Reply by zhongxianmen2020** [reply]
here is the link of the tutorial
https://pycaret.gitbook.io/docs/learn-pycaret/official-blog/time-series-101-for-beginners

**Reply by zhongxianmen2020** [reply]
I tried this tutorial on my local machine and on cloud which is Amazon Domino. I got the same thing.

**Reply by ReinvanLennep** [reply]
I think what @ngupta23 means is that when you run your setup function you get the option down below (scroll all the way down) to press enter or fill in the word "quit". For me I needed to click on the text bar and hit enter the first time. 

I hope this solved your problem, else maybe make a screenshot of your code.
